app/models/waste_exemptions_engine/charge_detail.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  class ChargeDetail < ApplicationRecord
+    self.table_name = "charge_details"
+
+    has_many :band_charge_details, dependent: :destroy
```

Comment:
Eventually yes, when we start to save charge and payment details on the order. I don't think it's required now however, as there is nothing in this PR to persist the charge_details. I expect that will be done by whatever calls the OrderCalculatorService. It might be best to wait for that change to introduce the association between order and charge_details, rather than me guessing now what will be needed later.